app-kotlin/src/main/res/layout/home_view_controller.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
```

Comment:
do we need a constraint layout?

Reply:
i had fixed this constraint layout by replacing all with frame layouts. in the latest pr #4